ssl/ssl_version_test.cc

```diff
@@ -2328,7 +2328,27 @@ TEST_P(SSLVersionTest, PeerTmpKey) {
     GTEST_SKIP();
   }
 
-  // Default should be using X5519 as the key exchange.
+
+  ASSERT_TRUE(Connect());
+  for (SSL *ssl : {client_.get(), server_.get()}) {
+    SCOPED_TRACE(SSL_is_server(ssl) ? "server" : "client");
+    EVP_PKEY *key = nullptr;
+
+    if (getVersionParam().version == TLS1_3_VERSION) {
+        // TLS 1.3 default should be using X5519MLKEM768 as the key exchange.
+        ERR_clear_error();
+        EXPECT_FALSE(SSL_get_peer_tmp_key(ssl, &key));
+        ErrorEquals(ERR_get_error(), ERR_LIB_SSL, SSL_R_UNKNOWN_KEY_EXCHANGE_TYPE);
```

Comment:
You have a good explanation for this in the PR overview:

> ssl_version_test.cc was updated to expect SSL_R_UNKNOWN_KEY_EXCHANGE_TYPE when calling SSL_get_peer_tmp_key() on an SSL connection that offered X25519MLKEM768. This is because there is no EVP_PKEY type for hybrid keys, only individual X25519 or MLKEM768 keys.

can we distill that into a source comment here?